### DISCUSSION TITLE
Lualine statusline disapear when running FZF / Floaterm?

### DISCUSSION THREAD

**Discussion by rodhash** [nvim-lualine/lualine.nvim#1068] [General]
Hi,

Is it expected to have the statusline disapearing when running FZF or Floaterm?

https://github.com/nvim-lualine/lualine.nvim/assets/29671981/d1de9903-ee94-4eb5-b2f2-3e48c7fe263b

I've added the extensions but it keeps disapearing.. when running Nvim-tree however lualine doesn't disapear.

```lua
  extensions = {'lazy', 'nvim-tree', 'toggleterm', 'fzf'},
```

I'm pretty sure in the past it didn't disapea, I tried rolling back few commits without luck.

Note:

If I enable tabline I get both indicators as expected .. however I rather have the statusline always visible, any idea?

https://github.com/nvim-lualine/lualine.nvim/assets/29671981/b804fba9-cebf-40cc-a1e8-6d74982062bb

**Comment by YUAN-ix** [comment]
neovim hides a statusline when opening a float window except a sidebar by default,so if you open a float neotree,it would disapear either,you will find it when you read the doc carefully
